Source/SpatialGDK/Private/SpatialActorChannel.cpp

```diff
@@ -331,8 +331,15 @@ bool USpatialActorChannel::ReplicateActor()
 	if (Binding)
 	{
 		uint32 ShadowDataOffset = 0;
+		uint32 MinAlignment = 1;
```

Comment:
int32

Reply:
what's the reason with using int over uint?